recipes/cdp/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cdp" %}
+{% set version = "1.3.3" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CDAT/cdp/archive/v{{ version }}.tar.gz
+  sha256: 8c3e69c695ecdf0c8e3d0fe38e4ae6987445aff104a89e837e496818ff9bb343
+
+build:
```

Comment:
Please include the python entry points:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

https://github.com/CDAT/cdp/blob/master/setup.py#L16